src/routes/release-build.js

```diff
@@ -50,15 +61,23 @@ router.get(
 
     const ciBuilds = [];
     if (build.ciBuilds) {
-      for (const circleBuild of build.ciBuilds.circle) {
+      for (const circleBuild of build.ciBuilds.circle ?? []) {
```

Comment:
I think we can actually kill this; CircleCI should be out of all branches on e/e now

Reply:
Yea, it might be safe to remove all together now, but part of me wants to leave it "just in case". 😅 A future PR could clean out dead code here.